src/common/utils/getPaneFooter.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  PaneFooter,
+} from '@folio/stripes/components';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getPaneFooter = (
```

Comment:
Could it be a functional component? Just to be consistent with our rest React architecture.

Reply:
Sure, maybe it's better to create in `stripes-acq-components` for all forms?

Reply:
Let's start from component in ui-invoice, to see if it leads us somewhere at all